docs/src/modules/spring/pages/getting-started.adoc

```diff
@@ -1 +1,170 @@
-= Getting Started
+= Getting started
+:page-aliases: spring:kickstart.adoc
+:page-aliases: spring:quickstart-java-maven.adoc
+
+include::ROOT:partial$include.adoc[]
+include::partial$attributes.adoc[]
+
+The Kalix development tools help you to get started quickly. They include:
+
+* A *Maven archetype* that generates the recommended project structure. The generated project is a regular Spring Boot project with a few configurations to integrate it seamlessly with Kalix.
+
+* A *Maven plugin* to help you in deploy your application to Kalix.
```

Comment:
```suggestion
* A *Maven plugin* to help you deploy your application to Kalix.
```

or `help you in deploying` I would say...

Reply:
Wondering if this comes from the java docs and needs to be fixed there as well or if I introduced it myself. I will check.